src/posix.mak

```diff
@@ -296,10 +296,11 @@ FRONT_SRCS=$(addsuffix .d, $(addprefix $D/,access aggregate aliasthis apply argt
 	dinifile dinterpret dmacro dmangle dmodule doc dscope dstruct dsymbol dsymbolsem	\
 	dtemplate dversion escape expression expressionsem func			\
 	hdrgen id impcnvtab imphint init initsem inline inlinecost intrange	\
-	json lib libelf libmach link mars mtype nogc nspace objc opover optimize parse permissivevisitor sapply templateparamsem	\
-	sideeffect statement staticassert target typesem traits transitivevisitor parsetimevisitor visitor	\
+	json lib libelf libmach link mars mtype nogc nspace objc opover optimize parse sapply templateparamsem	\
+	sideeffect statement staticassert target typesem traits \
 	typinf utils scanelf scanmach statement_rewrite_walker statementsem staticcond safe blockexit printast \
-	semantic2 semantic3))
+	semantic2 semantic3 \
+	$(addprefix visitors/, package parsetime permissive transitive)))
```

Comment:
What about `vistors.strict`? Why is it only part of the `PARSER_SRCS` list? (here and in win32.mak and the VS project).

Reply:
`visitors.strict` is a dead file - it isn't used by anything. I didn't want to modify the status quo in this PR.